src/transactions/OfferTests.cpp

```diff
@@ -1292,6 +1431,15 @@ TEST_CASE("create offer", "[tx][offers]")
                                           {xlm, usd, oneone, 110});
             });
         });
+        for_versions_from(10, *app, [&] {
+            auto offer = market.requireChangesWithOffer({}, [&] {
+                return market.addOffer(a1, {xlm, usd, oneone, 9});
+            });
+            REQUIRE_THROWS_AS(market.updateOffer(a1, offer.key.offerID,
+                                                 {xlm, usd, oneone, 111},
+                                                 {xlm, usd, oneone, 110}),
```

Comment:
it's a bit confusing to have an expected state different as it's a failure case

Reply:
Agreed, fixed.